client/src/Components/App.js

```diff
@@ -1,13 +1,15 @@
 import React, { Component } from 'react';
 import './App.css';
+import HeaderComp from './CommonComponents/Header';
```

Comment:
let's make the react components as groups like this
`import React, { Component } from 'react';
import HeaderComp from './CommonComponents/Header';

import './App.css';`

and make spaces between them